### PR TITLE
Added "deltavf2" lobby screen

### PR DESCRIPTION
**Pull request by ProspectorAres**
## About the PR
Added a lobby screen by the name "deltavf2", based off the game Team Fortress 2.

## Media

deltavf2

## Requirements

- [ x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x ] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Changed attribution.yml

**Changelog**

🆑 
- add: Added a new lobby screen named "deltavf2", based off the game Team Fortress 2.

### CODE REVIEW

File: Resources/Textures/LobbyScreens/attributions.yml [hunk 1]
```diff
@@ -57,3 +57,8 @@
   license: "Custom"
   copyright: "p.r.ospector on discord / Prospector @ DeltaV"
   source: "https://discord.gg/deltav"
+
+- files: ["deltavf2.png"]
+  lisence: "custom"
```
Comment (Colin-Tel) [inline Resources/Textures/LobbyScreens/attributions.yml:62]:
```suggestion
  license: "custom"
```

Comment (pissdemon) [reply Resources/Textures/LobbyScreens/attributions.yml:62]: Also, "Custom" doesn't tell anyone anything about how they are allowed to use the lobby screen. It's meaningless as a license without further info, so I don't think DV can make use of your contribution without more detail.

For simplicity's sake I suggest using [CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0) as license, or one of the [other CC licenses](https://creativecommons.org/share-your-work/cclicenses/), please read the latter page for details.

Comment (ProspectorAres) [reply Resources/Textures/LobbyScreens/attributions.yml:62]: Thank you for telling me, do I edit the PR?

Comment (deltanedas) [reply Resources/Textures/LobbyScreens/attributions.yml:62]: you can just push changes to the branch you dont have to close it

Comment (ProspectorAres) [reply Resources/Textures/LobbyScreens/attributions.yml:62]: I didn't know how to do that so I made a new PR.

**Pull request closed without merging**